### PR TITLE
Us3 add bookmark

### PR DESCRIPTION
**Pull request by clemensschueller**
Hi, Please take a look at my userstory. I added a bookmark page and a nav bar on top to switch between all shown cards and bookmarked. One can bookmark a favorite gallery on the details page.

### CODE REVIEW

**Review by roscha80 — APPROVED** [review]
Looks good for me. Good Work dude :)

File: src/App.js [hunk 1]
```diff
@@ -1,22 +1,39 @@
 import { useState } from 'react'
-import galleries from './data.json'
+import galleriesData from './data.json'
 import ListPage from './pages/ListPage'
 import DetailsPage from './pages/DetailsPage'
+import BookmarkPage from './pages/BookmarkPage'
+import Navigation from './components/Navigation'
 import { Route, Switch, useHistory } from 'react-router-dom'
```
Comment (frederikebruendermann) [inline src/App.js:7]: You could consider organizing your imports with cmd+shift+p :)

File: src/App.js [hunk 2]
```diff
@@ -1,22 +1,39 @@
...
+            onNavigate={handleClickBack}
+            handleBookmark={handleBookmark}
+            gallery={detailedGallery}
+          />
+        </Route>
+        <Route path="/bookmarks">
+          <BookmarkPage
+            onNavigate={handleClickDetails}
+            galleries={galleries}
+            // isBookmarked={isBookmarked}
```
Comment (frederikebruendermann) [inline src/App.js]: Why is this part commented out? :) Same in row 11.

File: src/App.js [hunk 3]
```diff
@@ -25,10 +42,23 @@ export default function App() {
...
+    const index = galleries.findIndex(gallery => gallery.id === id)
+    const selectedGallery = galleries[index]
+    selectedGallery.isBookmarked = !selectedGallery.isBookmarked
+    const updatedGalleries = [
+      ...galleries.slice(0, index),
+      selectedGallery,
+      ...galleries.slice(index + 1),
+    ]
+    setGalleries(updatedGalleries)
   }
```
Comment (frederikebruendermann) [inline src/App.js]: 🚀 Well done!

File: src/pages/DetailsPage.js [hunk 4]
```diff
@@ -14,11 +15,24 @@ DetailsPage.propTypes = {
...
+    name,
+    style,
+    address,
+    opening,
+    description,
+    isBookmarked,
+  } = gallery
   return (
     <Wrapper>
       <img src={image} alt="" />
```
Comment (frederikebruendermann) [inline src/pages/DetailsPage.js:31]: Maybe it would be good to use Cumulative Layout Shift at your images to improve the User Experience.

**Review by frederikebruendermann — APPROVED** [review]
Good work 👍🏼 Just some small suggestions from my side.

**Pull request merged**